frontend/src/app/auth/auth-routing.module.ts

```diff
@@ -3,13 +3,18 @@ import { Routes, RouterModule } from '@angular/router';
 
 import { SigninPage } from './pages/signin/signin.page';
 import { SignupPage } from './pages/signup/signup.page';
+import { ForgotPasswordPage } from './pages/forgot-password/forgot-password.page';
+import { ChangePasswordPage } from './pages/change-password/change-password.page';
+
 
 import { NoAuthGuard } from './guards/no-auth/no-auth.guard';
 
 const routes: Routes = [
   { path: '', redirectTo: 'signin', pathMatch: 'full' },
   { path: 'signin', component: SigninPage, canActivate: [ NoAuthGuard ] },
-  { path: 'signup', component: SignupPage, canActivate: [ NoAuthGuard ] }
+  { path: 'signup', component: SignupPage, canActivate: [ NoAuthGuard ] },
+  { path: 'forgotpassword', component: ForgotPasswordPage  },
+  { path: 'change-password', component: ChangePasswordPage }
```

Comment:
Let's use either both `forgotpassword` and `changepassword` or either `forgot-password` and change-password`.